core/includes/form.inc

```diff
@@ -2886,17 +2886,19 @@ function form_select_options($element, $choices = NULL) {
     }
     else {
       $key = (string) $key;
+      $option_attributes = array();
+      $option_attributes['value'] = check_plain($key);
       if ($value_valid && (!$value_is_array && (string) $element['#value'] === $key || ($value_is_array && in_array($key, $element['#value'])))) {
-        $selected = ' selected="selected"';
-      }
-      else {
-        $selected = '';
+        $option_attributes['selected'] = 'selected';
       }
 
       if (!empty($element[$key]['#indentation'])) {
         $choice = str_repeat('· ', $element[$key]['#indentation']) . $choice;
+        $option_attributes['data-indentation-depth'] = $element[$key]['#indentation'];
+        $option_attributes['class'] = 'indent';
```

Comment:
Maybe be consistent with indent/indentation here:
```suggestion
        $option_attributes['class'] = 'indentation';
```

Reply:
Fixed all 3 instances of this class with [`c275dc6` (#4381)](https://github.com/backdrop/backdrop/pull/4381/commits/c275dc628a1083ee94102d6ba56a7f80b3ce2502)